test/FourBitTwoDisclosureDeviceUnlockerTest.java

```diff
@@ -53,23 +74,47 @@ public void heuristicHaltingTestForUnlock() {
      * This should test for 99% efficiency in unlocking all 3 kinds of devices: linear, polynomial, and random rotations.
      */
     public void efficiencyTestForUnlock() {
-        int successes = 0;
-        for (int i = 0; i < 1000; i++) {
-            Device dev = null;
-            if (i % 3 == 0) dev = new Device();
-            else if (i % 3 == 1) dev = new Device(true);
-            else if (i % 3 == 2) dev = new Device(false, (int)(Math.random() * 4), (int)(Math.random() * 10));
-            if (FourBitTwoDisclosureDeviceUnlocker.unlock(dev)) successes++;
-            else System.out.println(dev + ": " + dev.superPeek());
+        Constructor empty = null;
```

Comment:
Again, this is just using reflection to access the now-private constructors and methods.